package.json

```diff
@@ -40,7 +40,6 @@
   "devDependencies": {
```

Comment:
can we bump the package minor version?

Reply:
I suggest doing this with npm before publishing. This also creates a tag. 

For example:

```
npm version minor
npm publish
```

You can also try using Sindre Sorhus' `np` to help you publish code safely, GitHub releases included.

Reply:
> This is all automated by GitHub actions

All good then! No need to change it here if it's all automated